src/Microsoft.ML.Transforms/FourierDistributionSampler.cs

```diff
@@ -183,9 +189,14 @@ public override float Next(Random rand)
     }
 
     /// <summary>
-    /// The Laplacian kernel is defined as k(x,y)=exp(-a*|x-y|_1). The distribution that is the Fourier transform of this
-    /// kernel is the Cauchy distribution with parameters (0, a).
+    /// The Laplacian kernel.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// The Laplacian kernel is defined as $k(x,y)=\exp(-a||x-y||)$. The distribution that is the Fourier transform of
```

Comment:
The second sentence is not quite clear to me.

Reply:
+1. please reword the second sentence about fourier transform.

---
In reply to: [277856329](https://github.com/dotnet/machinelearning/pull/3545#discussion_r277856329) [](ancestors = 277856329)